salt/modules/disk.py

```diff
@@ -191,6 +202,8 @@ def percent(args=None):
         cmd = 'df -P'
     elif __grains__['kernel'] == 'OpenBSD':
         cmd = 'df -kP'
+    elif __grains__['kernel'] == 'AIX':
+        cmd = 'df -kP'
```

Comment:
This appears to be the same statement as the conditional above it for `OpenBSD`. They could be combined.